src/preferencesControl.ts

```diff
@@ -284,6 +284,18 @@ export class PreferencesControl {
         }
 
         for (let radio of acceptRejectButtons) {
+            radio.addEventListener('blur', (event) => {
```

Comment:
Extract this into a separate function. Don't do everything in one place. This function (from its name) is intended to deal with prev/next focus control. And should not do anything else.